src/components/Accessories/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./style.css";
+import Product from "../Product";
+
+
+const Accessories = ({title}) => {
+    return (
+        <section class="accessories">
```

Comment:
jak jsme si řekli, mělo by se používat className místo class, viz třeba 
- https://stackoverflow.com/questions/46989454/class-vs-classname-in-react-16
- https://www.reddit.com/r/reactjs/comments/8j057o/can_you_use_class_instead_of_classname_in_react/
jakože funguje to, ale může se to někdy rozbít. PLus všichni ostatní používají className

Reply:
A jo vlastně! Díky za upozornění, přehlédla jsem to.